packages/expo-contacts/src/Contacts.ts

```diff
@@ -397,14 +399,24 @@ export type Contact = {
   isFavorite?: boolean;
 };
 
+/**
+ * Type for creating new contacts - excludes the id field since it will be generated by the OS.
+ */
+export type NewContact = Omit<Contact, 'id'>;
+
+/**
+ * Type for existing contacts returned from the system - guarantees the id field is present.
+ */
+export type ExistingContact = Contact & { id: string };
```

Comment:
Instead of omitting the `id`, can we move it from `Contact` to `ExistingContact`?